CODE/logicytics/Get.py

```
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+import configparser
+import os.path
+from pathlib import Path
+
+
+class Get:
+    @staticmethod
+    def list_of_files(directory: str, append_file_list: list = None,
```

Ruff (0.8.0)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)